platform-core-module/src/main/java/com/softicar/platform/core/module/program/IProgram.java

```diff
@@ -12,4 +12,9 @@ default IDisplayString toDisplay() {
 	}
 
 	void executeProgram();
+
+	default String getDefaultCronExpression() {
```

Comment:
should provide a Javadoc comment, especially concerning the possible results, e.g. `""`, `null`, etc.